pkg/gameservers/health_test.go

```diff
@@ -286,7 +288,44 @@ func TestHealthControllerSyncGameServer(t *testing.T) {
 	}
 }
 
+func TestHealthControllerSyncGameServer_UpdateFailed(t *testing.T) {
```

Comment:
Decided to put this case in a separate function in order to not to overload _TestHealthControllerSyncGameServer_ with extra _if_ statements to trigger and check an error.

Reply:
Minor nit: remove the underscore :smile: 

Otherwise - LGTM!

Reply:
This is my habit 😄